.github/workflows/test-build-deploy.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           bundle exec rake db:create
           bundle exec rake db:schema:load
 
+      - name: Brakeman
+        uses: reviewdog/action-brakeman@v2
+        with:
+          brakeman_version: 5.4.1
```

Comment:
Looks like you can set this to `gemfile` to use the version specified in the gemfile.